app/controllers/messages_controller.rb

```diff
@@ -21,4 +22,15 @@ def create
   def message_params
     params.require(:message).permit(:content)
   end
+
+  def notify_recipient
+    users_in_room = @message.joined_users
+    users_in_room.each do |user|
+      next if user.eql?(current_user)
+
+      notification = MessageNotification.with(message: @message.content, chatroom: @message.chatroom)
```

Comment:
creates Notification.

Reply:
Looks like it also skips notifying if it's from the current user?